asm/macros/battle_script.inc

```diff
@@ -1674,14 +1674,51 @@
 	callnative BS_FickleBeamDamageCalculation
 	.endm
 
-@ various command changed to more readable macros
-	.macro cancelmultiturnmoves battler:req
-	various \battler, VARIOUS_CANCEL_MULTI_TURN_MOVES
+	.macro jumpifteainvulnerable battler:req, jumpInstr:req
+	callnative BS_TeatimeInvul
+	.byte \battler
```

Comment:
the battler argument can be removed and replaced in the function with `gBattlerTarget`